src/Compatibility/Core/src/Android/Forms.cs

```diff
@@ -97,9 +98,11 @@ public static Color GetColorButtonNormal(Context context)
 			return _ColorButtonNormal;
 		}
 
+		[Obsolete]
 		public static void Init(IActivationState activationState, InitializationOptions? options = null) =>
 			Init(activationState.Context, options);
```

Comment:
I marked all of the Init things in the Forms type obsolete so that everyone knows not to use it. It is still used internally for now, but this is basically a non-public thing with all the work for the hosting and registration.